File: src/main/java/org/shark/example/controller/mysql/pojo/JsonInnerJoinDto.java
package org.shark.example.controller.mysql.pojo;

import lombok.Data;
import org.shark.example.dao.repository.pojo.JsonExampleDo;
import org.shark.example.service.mysql.pojo.CreateJsonExampleDto;

@Data
public class JsonInnerJoinDto {
    private Integer id;
    private CreateJsonExampleDto createJsonExampleDto;
    private String description;
}

File: src/main/kotlin/org/shark/example/controller/mysql/pojo/JsonInnerJoinDto.kt
package org.shark.example.controller.mysql.pojo

import org.shark.example.service.mysql.pojo.CreateJsonExampleDto

data class JsonInnerJoinDto(
    var id: Int? = null,
    var createJsonExampleDto: CreateJsonExampleDto? = null,
    var description: String? = null
)
